locales/en-US.json

```
@@ -220,6 +220,7 @@
     "View Reddit comments": "View Reddit comments",
     "Hide replies": "Hide replies",
     "Show replies": "Show replies",
+    "Replies loaded": "loaded",
```

Maybe this should be "replies loaded" instead of "loaded"? 

```suggestion
    "Replies loaded": "replies loaded",
```